app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -166,6 +166,8 @@ export class AwsServerGroupConfigurationService {
       command.suspendedProcesses.includes(process);
 
     cmd.onStrategyChange = (command: IAmazonServerGroupCommand, strategy: IDeploymentStrategy): void => {
+      command.setLaunchTemplate = strategy.key === 'rollingpush' ? true : undefined;
```

Comment:
I am hoping we don't need this. It feels very specific to rollingPush.

Reply:
The tricky thing is that rollingpush is very orca heavy. Let me play around with this branch a bit.

Reply:
Just let me know. For now its only useful for rollingPush, but we can always take it out if we don't end up needing this.

Reply:
Sounds good!